app/src/main/java/com/globant/example/mentorapp/data/entity/EventApiResponseEntity.java

```diff
@@ -0,0 +1,37 @@
+package com.globant.example.mentorapp.data.entity;
+
+/**
+ * Class to manage otto responses from ApiClient
+ * Created by oscar.amezquita on 13/06/2017.
+ */
+
+public class EventApiResponseEntity {
```

Comment:
This event is the way of conveying data from interactors to presenters. It should also contain the result. You can add an object T that represents the body or response, so this event can be easily scalable.